requirements/common/constraints.txt

```diff
@@ -61,3 +61,5 @@ langchain-community>=0.2.5
 # NOTE(robinson): choma was pinned to importlib-metadata>=7.1.0 but 7.1.0 was installed
 # instead of 7.2.0. Need to investigate
 importlib-metadata==7.1.0
+
+nltk==3.8.1
```

Comment:
this seems unrelated. Is it limited by the pydantic version?

Reply:
This was due to an update in unstructured that I think it going ton be fixed in the next release.